packages/app/src/manifest.json

```diff
@@ -8,26 +8,18 @@
     "256": "assets/connect-logo/Stacks256w.png",
     "512": "assets/connect-logo/Stacks512w.png"
   },
-  "content_security_policy": "script-src 'self'<% DEV_CSR %>; object-src 'none'; frame-src 'none'; frame-ancestors 'none';",
-  "permissions": [
-    "activeTab",
-    "storage"
-  ],
+  "content_security_policy": "script-src 'self'<% DEV_CSR %>; object-src <% DEV_OBJECT_SRC %>; frame-src 'none'; frame-ancestors 'none';",
```

Comment:
@hstove thoughts?

Reply:
Yeah if it doesn't break anything, this seems good. I'm guessing this would block things like externally-hosted fonts (which we don't want anyways), right?